timemachine/fe/utils.py

```diff
@@ -148,6 +149,16 @@ def get_romol_conf(mol):
     return guest_conf / 10  # from angstroms to nm
 
 
+def set_romol_conf(mol, new_coords):
+    """Sets coordinates of mol's 0th confomrmer"""
+    assert new_coords.shape[0] == mol.GetNumAtoms()
+    # convert from nm to angstroms
+    angstrom_coords = new_coords * 10
```

Comment:
Should we indicate in the function name and/or the docstring that this also performs unit conversion?

Reply:
Done in [5ab4c26](https://github.com/proteneer/timemachine/pull/813/commits/5ab4c26e0267f6ba02fee9a8f05a3764deb3c81f)